ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/deprovision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-deprovision
+  steps:
+  - ref: gather-aws-console
+  - chain: gather
+  - ref: gather-proxy
+  - ref: ipi-deprovision-deprovision
```

Comment:
I am trying to tidy up deprovision chains in https://github.com/openshift/release/pull/33289, so suggest to wrapper the above lines with `chain: cucushift-installer-rehearse-aws-ipi-deprovision`